agent/lib/src/external_api/model.rs

```diff
@@ -0,0 +1,139 @@
+use serde::{Deserialize, Serialize};
+
+#[derive(Deserialize, Serialize, Debug, PartialEq, Eq)]
+pub struct FileModel {
+    pub filename: String,
+    pub content: String,
```

Comment:
Is `content` the content of the file at `filename`? Is `filename` a path? or just a name?

Reply:
It is a path relative to the workspace directory in ``internal::service``